src/libraries/System.Console/src/System/TerminalFormatStrings.cs

```diff
@@ -51,8 +51,7 @@ internal sealed class TerminalFormatStrings
     /// The dictionary of keystring to ConsoleKeyInfo.
     /// Only some members of the ConsoleKeyInfo are used; in particular, the actual char is ignored.
     /// </summary>
-    public readonly Dictionary<ReadOnlyMemory<char>, ConsoleKeyInfo> KeyFormatToConsoleKey =
-        new Dictionary<ReadOnlyMemory<char>, ConsoleKeyInfo>(new ReadOnlyMemoryContentComparer());
+    public readonly Dictionary<string, ConsoleKeyInfo> KeyFormatToConsoleKey = new();
```

Comment:
Presumably this needs to be passed an explicit comparer for alternate lookup to work.